lib/spack/spack/relocate.py

```diff
@@ -351,6 +351,19 @@ def relocate_binary(path_names, old_dir, new_dir, allow_root):
         tty.die("Relocation not implemented for %s" % platform.system())
 
 
+def make_link_relative(cur_path_names, orig_path_names):
+    """
+    Change absolute links to be relative to old_dir
```

Comment:
What is `old_dir` here?

Reply:
I think that's a remnant of an older version of the function signature. Deleting.